src/utils/constants.py

```diff
@@ -0,0 +1,50 @@
+from enum import Enum, auto
+
+
+class ChoicesEnum(Enum):
+    """Augments a regular enum with methods to return lists of enum
+    choices/names."""
+
+    @classmethod
+    def get_names(cls):
+        names = []
+        for choice in cls:
+            names.append(choice.name)
+
+        return names
+
+    @classmethod
+    def get_choices(cls):
+        choices = []
+        for choice in cls:
+            choices.append(choice)
+
+        return choices
+
+
+class Requests(ChoicesEnum):
+    START = auto()
+    STARTING = auto()
+    STOP = auto()
+    STOPPING = auto()
+    ARE_YOU_ALIVE = auto()
+    YES = auto()
+
+
+class RunLevels(ChoicesEnum):
+    LOCAL = auto()
+    LOCAL_STANDALONE = auto()
+    PRODUCTION = auto()
+
+
+class LogLevels(ChoicesEnum):
```

Comment:
unnecessary enum, the python logging module already provides methods necessary for translating a string choice to a log.level, its in the old code

Reply:
Maybe, but it's nice to use the `get_names()` method while setting up logging to prevent some string typo.

Reply:
This gives us the ability to call `LogLevels.choices()`, saving us from having to write strings in a few places.